test/unit/services/platform-test.js

```diff
@@ -74,4 +75,27 @@ describe('Platform', function() {
 
   });
 
+  describe('isVirtualizationEnabled', function(){
+
+    it('should able to check for mac', function() {
```

Comment:
should be 
```JavaScript
describe('on mac', function() {
  it('should return true if virtualization check shell script returns true');
});
```
'it' function without callback is called 'pending' test, in is fine for macOS for now.